### PR TITLE
Use nltk to tokenize text

### PR DESCRIPTION
**Pull request by titipata**
I change the function so that it uses `nltk` to filter stop words and tokenize words instead so that we don't have to provide `csv` file of English stop words.

### CODE REVIEW

File: backend_python_scripts/text_analysis_functions.py [hunk 1]
```diff
@@ -20,49 +17,32 @@ def scrape_frontpage_terms(page='http://www.politifact.com',stopwords_csv=None):
...
-    text = [chunk.split() for chunk in chunks if chunk]
-
-    cleaned_words = []
-    for words in text:
-        for word in words:
-            word = word.lower()
-            word = filter(lambda x: x.isalnum(),word)
-            if word not in stopwords:
-                cleaned_words.append(word)
+    text_tokenize = word_tokenize(text_replace)
```
Comment (torbenator) [inline backend_python_scripts/text_analysis_functions.py:33]: text_replace isn't defined. I think this just got left over from the line above. It works fine if I comment this out.

File: backend_python_scripts/text_analysis_functions.py [hunk 2]
```diff
@@ -74,35 +54,23 @@ def filter_wild_text(sentence,textbank):
...
-
-    #Headline on cnn.com lol
-    raw_text = "17 years of interviews reveal trump's penchant for lewd talk"
-
-
-    words_to_search = filter_wild_text(raw_text,text_bank)
-    print words_to_search
-
-
+    text_bank = build_text_bank(stopwords_csv=True) # make a bank of words using politifact's front bage
```
Comment (torbenator) [inline backend_python_scripts/text_analysis_functions.py:73]: just had to change stopwords_csv to stop_words

**Review by torbenator — APPROVED** [review]
Awesome edits! This is much more pythonic solution and its a lot easier to read. Thanks dude 👍

File: backend_python_scripts/text_analysis_functions.py (hunk 2 above)
Comment (titipata) [reply backend_python_scripts/text_analysis_functions.py:73]: Thanks @torbenator for catching the bugs!

**Pull request merged**